### ISSUE TITLE
When loading vae model train_device_ attribute is not set for SOLO.from_scvi_model

### ISSUE THREAD

**Issue by njbernstein** [YosefLab/scvi-tools#1069]
When loading a scvi model and then creating a solo model from that scvi model we get an error because train_device_ is not set.

```python
# doesnt work
solo = SOLO.from_scvi_model(vae_loaded)
# works
from scvi.model._utils import parse_use_gpu_arg
_, b = parse_use_gpu_arg(False)
vae_loaded.train_device_ = b
solo = SOLO.from_scvi_model(vae_loaded)
```

```pytb
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-52-98c138298a07> in <module>
----> 1 solo = SOLO.from_scvi_model(vae_loaded)

~/repos/scvi-tools/scvi/external/solo/_model.py in from_scvi_model(cls, scvi_model, adata, restrict_to_batch, doublet_ratio)
    160         # get latent representations and make input anndata
    161         latent_rep = scvi_model.get_latent_representation(
--> 162             orig_adata, indices=batch_indices
    163         )
    164         lib_size = scvi_model.get_latent_library_size(

~/miniconda3/envs/solo-sc/lib/python3.7/site-packages/torch/autograd/grad_mode.py in decorate_context(*args, **kwargs)
     25         def decorate_context(*args, **kwargs):
     26             with self.__class__():
---> 27                 return func(*args, **kwargs)
     28         return cast(F, decorate_context)
     29 

~/repos/scvi-tools/scvi/model/base/_vaemixin.py in get_latent_representation(self, adata, indices, give_mean, mc_samples, batch_size)
    156         if self.is_trained_ is False:
    157             warnings.warn("Model hasn't been trained yet, results will be random.")
--> 158         elif self.device.type != self.train_device_.type:
    159             warnings.warn(
    160                 f"""Model trained with {self.train_device_} but currently on {self.device}. You may experience

AttributeError: 'NoneType' object has no attribute 'type'
```

#### Versions:

> VERSION

<img width="1146" alt="Screen Shot 2021-05-19 at 3 41 58 PM" src="https://user-images.githubusercontent.com/6864886/118894276-c398db00-b8b8-11eb-8714-a06f4416ea66.png">

**Comment by adamgayoso** [comment]
was `vae_loaded` a model loaded from a different version?

**Comment by adamgayoso** [comment]
this was also just fixed in #1070 so shouldn't be an issue now if you pull again
